middleware/markdown/markdown_test.go

```diff
@@ -163,11 +150,9 @@ DocFlags.var_bool true`
 		<title>Markdown test 2</title>
 		<meta charset="utf-8">
```

Comment:
https://github.com/mholt/caddy/pull/762/files#diff-e6fcb404819a44598c821bf0c4da0dbdR146

Can be sorted later IMHO.